modules/src/main/java/eu/itesla_project/modules/contingencies/ContingenciesAndActionsDatabaseClientFactory.java

```diff
@@ -25,4 +26,8 @@ default ContingenciesAndActionsDatabaseClient create(Path contingenciesAndAction
         return create();
     }
 
+    @Override
+    default ContingenciesAndActionsDatabaseClient create(InputStream data) {
```

Comment:
Indeed! Default implementation are provided by ContingenciesProviderFactory. The difference here is the return type.